generate_seq_signal_node:
  ros__parameters:
    deltacan_topic: "/deltacan"
    control_freq: 50.0
    noise: 0.0
    sequence:
    - "1.0, 0.0, 5.0, 0.0, 1.0"
    - "1.0, 0.0, 5.0, -1.0, 0.0"
    - "1.0, 0.0, 10.0, 0.0, 1.0"
    - "1.0, 0.0, 10.0, -1.0, 0.0"
    - "1.0, 0.0, 15.0, 0.0, 1.0"
    - "1.0, 0.0, 15.0, -1.0, 0.0"
    - "1.0, 0.0, 20.0, 0.0, 1.0"
    - "1.0, 0.0, 20.0, -1.0, 0.0"
    - "1.0, 0.0, 30.0, 0.0, 1.0"
    - "1.0, 0.0, 30.0, -1.0, 0.0"
    pause_duration: 5.0
